UPGRADE.md

```diff
@@ -237,6 +243,62 @@ INSERT INTO ca_category_translation_medias (idCategoryTranslations, idMedia) SEL
 DROP TABLE category_translation_media_interface;
 ```
 
+### Routing changes
+
+Some resources in app/config/admin/routing.yml need to be removed:
+- sulu_tag
+- sulu_contact
+- sulu_category
+- sulu_custom_urls_api 
```

Comment:
Think this should be `sulu_content`.

I'm wondering if there are not more as I think there should now be only apis. /cc @danrot

Reply:
I'm sorry, it should be the list as in https://github.com/sulu/sulu-minimal/compare/master...develop (I think)

Reply:
Not really, since there are still a few non-API routes. I have only removed the ones which were generating the templates for our forms. There are still endpoints not being an API e.g. for generating images.

Reply:
@danrot ok most of them seems to be api's like cache:clear or location, ... 

but think the following we could maybe remove:

 - sulu_media ( are just the form templates )
 - sulu_media_website ( was only here for the old preview I think? )
 - sulu_search ( was only here for the old preview I think? )

but thats now not related to this PRs.

@reyostallenberg think the only missing thing we had is the PHP 7.1 requirement which should also be added.